### PR TITLE
sys-fs/cryfs: Add version bump to 0.10.0

### PR DESCRIPTION
**Pull request by linxon**
Bug: https://bugs.gentoo.org/678572
Package-Manager: Portage-2.3.62, Repoman-2.3.11

### CODE REVIEW

**Comment by Dessa** [comment]
is there a reason for this not being 0.10.1?

**Comment by linxon** [comment]
>is there a reason for this not being 0.10.1?

`0.10.1` require `>=dev-libs/boost-1.65.1`

```
...
-- Boost will be dynamically linked
CMake Error at /usr/share/cmake/Modules/FindBoost.cmake:1942 (message):
  Unable to find the requested Boost libraries.

  Boost version: 1.65.0

  Boost include path: /usr/include

  Detected version of Boost is too old.  Requested version was 1.65.1 (or
  newer).
Call Stack (most recent call first):
  cmake-utils/utils.cmake:119 (find_package)
  src/blockstore/CMakeLists.txt:38 (target_add_boost)
...
```

File: sys-fs/cryfs/cryfs-0.10.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (a17r) [inline sys-fs/cryfs/cryfs-0.10.0.ebuild]: why not 7?

File: sys-fs/cryfs/cryfs-0.10.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+CMAKE_MAKEFILE_GENERATOR=emake
```
Comment (a17r) [inline sys-fs/cryfs/cryfs-0.10.0.ebuild]: is that still necessary? also, one empty newline after EAPI please.

File: sys-fs/cryfs/cryfs-0.10.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+	sed -i -e '/WARNING! This is a debug build. Performance might be slow./d' src/cryfs-cli/Cli.cpp || die
+}
+
+src_configure() {
+	# upstream restricts installing files to Release configuration
+	# (CMAKE_BUILD_TYPE does not affect anything else)
+	local CMAKE_BUILD_TYPE
+	local -a mycmakeargs
+	CMAKE_BUILD_TYPE=Release
+	mycmakeargs=(
```
Comment (a17r) [inline sys-fs/cryfs/cryfs-0.10.0.ebuild]: local `mycmakeargs` makes line 53 obsolete.

Comment (automorphism88) [reply sys-fs/cryfs/cryfs-0.10.0.ebuild]: If you define a local variable and assign it on the same line, bash assigns the variable first and then marks it as local, so the variable remains set after leaving the function even though any further changes will be local. Try it out with a test script. This is why it's written this way, as with other areas where the `local` builtin is used. This differs from the behavior of some other shells.

**Comment by linxon** [comment]
Moved to: https://github.com/gentoo/gentoo/pull/11980

**Pull request closed without merging**